app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  attribute :text, :text
+
+  # Callbacks
+  after_create :update_post_comments_counter
+  after_destroy :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.update(comments_count: post.comments.count)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you make this method private in keeping with encapsulation and to avoid other parts of the app from calling this method since it is local to this class.

Reply:
done